src/paperbot/api/routes/sandbox.py

```
@@ -354,3 +362,194 @@ async def get_system_status(http_request: Request):
+    """Response for E2B settings"""
+    configured: bool
+    masked_key: Optional[str] = None
+
+
+def _load_settings() -> Dict[str, Any]:
+    """Load settings from file."""
+    import json
+    if SETTINGS_FILE.exists():
+        try:
+            return json.loads(SETTINGS_FILE.read_text())
+        except Exception:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)